libs/designsystem/modal/src/modal-wrapper/modal-wrapper.component.ts

```diff
@@ -350,6 +350,8 @@ export class ModalWrapperComponent
    * when ionScroll emits.
    */
   private initializeContentScrollListening() {
+    if (this.platform.isPhabletOrBigger()) return;
```

Comment:
Would be great to find a way to do this dynamically if the screen resizes, to satisfy the request in the issue: 

> Ideally we want to listen to the current breakpoint and enable/disable the subscription accordingly.